ibm/service/vpc/data_source_ibm_is_backup_policy.go

```diff
@@ -237,13 +246,15 @@ func dataSourceIBMIsBackupPolicyRead(context context.Context, d *schema.Resource
 		}
 	}
 
-	matchResourceType := make([]string, 0)
-	if backupPolicy.MatchResourceTypes != nil {
-		for _, matchResourceTyp := range backupPolicy.MatchResourceTypes {
-			matchResourceType = append(matchResourceType, matchResourceTyp)
+	if backupPolicy.MatchResourceType != nil {
+		d.Set("match_resource_types", []string{*backupPolicy.MatchResourceType})
```

Comment:
Keep this attribute but also add a new `match_resource_type` map value and add in documentation for deprecation of `match_resource_types` in data sources and to utilise `match_resource_type`